### PR TITLE
Add rebar parser

### PR DESCRIPTION
**Pull request by timgluz**
Hi, this PR adds support for Erlang Rebar3 package manager; (spec: http://www.rebar3.org/docs/dependencies)

It's was big task due to the fact that `rebar.config` uses erlang code for format and there was not code parser for that; 

It can reads dependencies from the `:deps` block and from each `:deps` block under each profile;

### CODE REVIEW

File: lib/versioneye/parsers/nuget_parser.rb [hunk 1]
```diff
@@ -317,7 +317,6 @@ def parse_dependency_version( project, dependency )
 
     project.projectdependencies.push(dependency)
     project.out_number     += 1 if ProjectdependencyService.outdated?( dependency )
-    project.unknown_number += 1 if product.nil?
```
Comment (reiz) [inline lib/versioneye/parsers/nuget_parser.rb]: @timgluz Why did you remove that? And why is this related to Erlang? That's the Nuget parser.

**Comment by reiz** [comment]
@timgluz I left 1 comment in the code, but otherwise it looks OK.

**Comment by timgluz** [comment]
It is double counting,  check couple of lines up. And you don't like #todo
blocks, so I removed it

File: lib/versioneye/parsers/nuget_parser.rb (hunk 1 above)
Comment (timgluz) [inline lib/versioneye/parsers/nuget_parser.rb]: It's double counting - exact same line is 5 lines up; i would add `#TODO:` comment, but you dont like those; then i had no other options than just remove it

Comment (reiz) [inline lib/versioneye/parsers/nuget_parser.rb]: Ah, now I see it. Looks like bug. Good job!
